CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Genesis Sample Theme Changelog
 
+## [3.4.2] - 2021-12-19
```

Comment:
```suggestion
## [3.4.2] - 2022-12-19
```

Reply:
@kienstra 😆 And, it is almost 2023...

Reply:
Haha, once I released a version of GCB with the wrong year